projects/bp-gallery/src/components/views/tag-table/TagTableView.scss

```diff
@@ -1,11 +1,8 @@
 .tag-grid {
   width: 100%;
+  height: calc(100% - 2rem);
 
   .merge-button {
     height: 2rem;
   }
-
-  .table {
-    height: calc(100% - 2rem);
-  }
```

Comment:
> The commit message ([c401ea6](https://github.com/hpi-swa-lab/BP2021RH1/commit/c401ea6e6ce671a668906cafe63300795bbc849a)) says "fix tag table grid". What exactly does this fix?

One of the css changes completely broke the tag table grid, so that it had a height of 0 and you couldn't curate any tags. I wouldn't even have noticed this if 2 of the cypress test hadn't failed.